charts/prometheus-operator-admission-webhook/Chart.yaml

```diff
@@ -2,9 +2,10 @@
 apiVersion: v2
 description: Prometheus Operator Admission Webhook
 name: prometheus-operator-admission-webhook
-version: 0.2.1
+version: 0.2.2
```

Comment:
I would suggest bumping to minor version since the labels get changed. Also please provide something in README.md that the users need to know for upgrading.

Reply:
Thank you, will do.